packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -168,7 +168,7 @@ export const PersonalDetailsForm = ({
         return () => {
             clearTimeout(timeout_id);
         };
-    }, [start_on_submit_timeout.is_timeout_started]);
```

Comment:
please add this to qa checklist to make sure that its working as before

Reply:
sure ako